### PR TITLE
Available command 

### LINKED ISSUE #75: Command to show which TAs are available for the given tutorial slot

### PR DESCRIPTION
**Pull request by yongkotaro**
Find available TA's using specific tutorial group. Input for tutorial tag must be exactly the same as the desired tutorial group and command only works for one tutorial group.

- Command input: available /g [TUTORIAL_GROUP]
- Closes #75

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AvailableCommand.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import seedu.address.model.tag.TutorialTag;
+
+/**
+ * Finds and lists all available TAs for the tutorial group.
+ */
+public class AvailableCommand extends Command {
+    public static final String COMMAND_WORD = "available";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all available TAs for the tutorial slot.\n"
+            + "Parameters: [FLAG] [TUTORIAL SLOT] \n"
```
Comment (wongkj12) [inline src/main/java/seedu/address/logic/commands/AvailableCommand.java:19]: For consistency with other commands should change to
"Parameters: available /g TUTORIAL"
(using COMMAND_WORD and PREFIX_GROUP)

**Review by wongkj12 — APPROVED** [review]
Minor issue in parameter description, but otherwise LGTM.

File: src/main/java/seedu/address/logic/commands/AvailableCommand.java (hunk 1 above)
Comment (yongkotaro) [reply src/main/java/seedu/address/logic/commands/AvailableCommand.java]: Thank you will take note of that 👍

File: src/main/java/seedu/address/logic/commands/AvailableCommand.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import seedu.address.model.tag.TutorialTag;
+
+/**
+ * Finds and lists all available TAs for the tutorial group.
+ */
+public class AvailableCommand extends Command {
+    public static final String COMMAND_WORD = "available";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all available TAs for the tutorial slot.\n"
+            + "Parameters: [FLAG] [TUTORIAL] \n"
```
Comment (yyccbb) [inline src/main/java/seedu/address/logic/commands/AvailableCommand.java:19]: should be PREFIX_GROUP + "TUTORIAL_GROUP"

**Review by yyccbb — APPROVED** [review]

**Pull request merged**